packages/react/src/components/Masthead/HeaderNavContainer.js

```diff
@@ -155,25 +221,49 @@ const HeaderNavContainer = ({ children }) => {
    */
   const handleOnKeyDown = event => {
     if (matches(event, [keys.Tab])) {
-      if (event.shiftKey) {
-        //Focus previous input
-        if (
-          document.activeElement.parentElement.previousSibling &&
-          document.activeElement.parentElement.previousSibling.offsetLeft +
-            position <=
-            buttonSize
-        ) {
-          paginateLeft();
+      if (pageIsRTL) {
+        if (event.shiftKey) {
+          if (
+            document.activeElement.parentElement.previousSibling &&
+            document.activeElement.parentElement.previousSibling.offsetLeft +
+              document.activeElement.parentElement.previousSibling.offsetWidth -
```

Comment:
just one question, would we be able to use optional chaining here and also on L238/L240?

Reply:
sure, I can add that